tests/models/whisper/test_modeling_flax_whisper.py

```diff
@@ -387,6 +388,19 @@ def test_save_load_to_base(self):
                     max_diff = (base_params[key] - base_params_from_head[key]).sum().item()
                     self.assertLessEqual(max_diff, 1e-3, msg=f"{key} not identical")
 
+    def test_encoder_sinusoidal_embed_positions(self):
```

Comment:
To test that the Flax embeddings are non-trainable (frozen), you can follow this Flax Wav2Vec2 test: https://github.com/huggingface/transformers/blob/4b4c6aabfb054fd42e5737ffafc56739adf3cce4/tests/models/wav2vec2/test_modeling_flax_wav2vec2.py#L281

To me this is optional: we know that the embeddings are initialised correctly through your test, and that grads are set to zero by action of `jax.lax.stop_gradient`, so up to you if you want to add this!